board.py

```diff
@@ -1,11 +1,31 @@
 from pieces import *
+from ascii import *
 
```

Comment:
In general, you will want to used named rather than starred imports to be explicit about what you rely on from other files.

Reply:
Got it. I was hoping that since the files I'm importing are my own files rather than someone else's that it was more licit. I need to... better understand how to use the import command in python. Will add to the list.